plugin/cscope_maps.vim

```diff
@@ -41,12 +41,40 @@ if has("cscope")
     " if you want the reverse search order.
     set csto=0
 
-    " add any cscope database in current directory
-    if filereadable("cscope.out")
-        cs add cscope.out
-    " else add the database pointed to by environment variable
-    elseif $CSCOPE_DB != ""
+    " cscope seems to not index files in symlinked directories.
```

Comment:
Can you please extract this detailed description into commit message? I think it would be better to have some short comment here in the code, something like:

    " Handle 'cs add cscope.out' correctly when following symlinks

I just feel that the detailed description belongs to commit message.